### PR TITLE
Update renamed or deleted files in dev-1.14-ko.5

### LINKED ISSUE #14582: Outdated files in the dev-1.14-ko.5 branch
**This is a Bug Report**

**Problem:**
Outdated files in the dev-1.14-ko.5 branch.

133 files in content/en are changed
21 files in content/ko are outdated

- [x]  _index.html 1 file changed, 2 insertions(+), 2 deletions(-)
- [x]  docs/concepts/_index.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x]  docs/concepts/architecture/cloud-controller.md 1 file changed, 4 insertions(+), 4 deletions(-)
- [x]  docs/concepts/overview/_index.md 1 file changed, 1 insertion(+), 2 deletions(-)
- [x]  docs/concepts/overview/object-management-kubectl/_index.md 1 file changed, 5 deletions(-)
- [x]  docs/concepts/workloads/pods/init-containers.md 1 file changed, 5 insertions(+), 4 deletions(-)
- [x]  docs/concepts/workloads/pods/pod-lifecycle.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x]  docs/concepts/workloads/pods/pod-overview.md 1 file changed, 12 insertions(+), 9 deletions(-)
- [x]  docs/reference/glossary/docker.md 1 file changed, 6 insertions(+), 7 deletions(-)
- [ ]  docs/setup/_index.md 1 file changed, 88 insertions(+), 73 deletions(-)
- [x]  docs/setup/certificates.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [ ]  docs/setup/minikube.md 1 file changed, 124 insertions(+), 97 deletions(-)
- [x]  docs/setup/multiple-zones.md 1 file changed, 2 insertions(+), 2 deletions(-)
- [x]  docs/setup/pick-right-solution.md 1 file changed, 328 deletions(-)
- [x]  docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md 1 file changed, 3 insertions(+), 3 deletions(-)
- [x]  docs/tasks/tools/install-minikube.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x]  docs/tutorials/configuration/configure-redis-using-configmap.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [ ]  docs/tutorials/online-training/overview.md 1 file changed, 23 insertions(+), 5 deletions(-)
- [x]  docs/tutorials/stateful-application/cassandra.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x]  examples/application/guestbook/redis-slave-deployment.yaml 1 file changed, 1 insertion(+), 1 deletion(-)
- [x]  examples/pods/config/redis-pod.yaml 1 file changed, 3 insertions(+)

4 files to be renamed
1. [x] R095 content/en/docs/concepts/overview/object-management-kubectl/overview.md -> content/en/docs/concepts/overview/working-with-objects/object-management.md
1. [x] R096 content/en/docs/concepts/overview/object-management-kubectl/declarative-config.md -> content/en/docs/tasks/manage-kubernetes-objects/declarative-config.md
1. [x] R089 content/en/docs/concepts/overview/object-management-kubectl/imperative-command.md -> content/en/docs/tasks/manage-kubernetes-objects/imperative-command.md
1. [x] R083 content/en/docs/concepts/overview/object-management-kubectl/imperative-config.md -> content/en/docs/tasks/manage-kubernetes-objects/imperative-config.md

**Proposed Solution**
Use `git diff` to check what is changed in the upstream. And apply the upstream changes manually to the `content/ko` of `dev-1.14-ko.5` branch.

For example:
```
# checkout `k/website:dev-1.14-ko.5`
...
# check what is updated in the upstream 
git diff dev-1.14-ko.4 dev-1.14-ko.5 -- content/en/docs/concepts/_index.md
# apply changes to content/ko
vi  content/ko/docs/concepts/_index.md
...
# commit and push
...
# make PR to `k/website:dev-1.14-ko.5`
```

**Page to Update:**
https://kubernetes.io/...

### PR DESCRIPTION
**Pull request by gochist**
fixes_ #14582 partly

### CODE REVIEW

File: content/ko/docs/tasks/manage-kubernetes-objects/declarative-config.md [hunk 1]
```diff
@@ -23,17 +31,17 @@ weight: 40
 * 명령형 오브젝트 구성
 * 선언형 오브젝트 구성
 
-오브젝트 관리 방식의 종류별 장단점에 대한 논의는 [Kubernetes Object Management](/docs/concepts/overview/object-management-kubectl/overview/)를
+오브젝트 관리 방식의 종류별 장단점에 대한 논의는 [Kubernetes Object Management](/ko/docs/concepts/overview/working-with-objects/object-management/)를
```
Comment (seokho-son) [inline content/ko/docs/tasks/manage-kubernetes-objects/declarative-config.md]:
```suggestion
오브젝트 관리 방식의 종류별 장단점에 대한 논의는 [쿠버네티스 오브젝트 관리](/ko/docs/concepts/overview/working-with-objects/object-management/)를
```

File: content/ko/docs/tasks/manage-kubernetes-objects/imperative-command.md [hunk 2]
```diff
@@ -10,7 +10,14 @@ weight: 20
 이를 사용하여 활성 오브젝트를 어떻게 관리하는 지에 대해 설명한다.
 {{% /capture %}}
 
-{{% capture body %}}
+{{% capture prerequisites %}}
+[`kubectl`](/docs/tasks/tools/install-kubectl/)를 설치한다.
```
Comment (seokho-son) [inline content/ko/docs/tasks/manage-kubernetes-objects/imperative-command.md]:
```suggestion
[`kubectl`](/docs/tasks/tools/install-kubectl/)을 설치한다.
```

File: content/ko/docs/tasks/manage-kubernetes-objects/imperative-command.md [hunk 3]
```diff
@@ -155,8 +162,8 @@ kubectl create --edit -f /tmp/srv.yaml
 {{% /capture %}}
 
 {{% capture whatsnext %}}
-- [오브젝트 구성을 이용하여 쿠베네티스 관리하기(명령형)](/docs/concepts/overview/object-management-kubectl/imperative-config/)
-- [오브젝트 구성을 이용하여 쿠버네티스 관리하기(선언형)](/docs/concepts/overview/object-management-kubectl/declarative-config/)
-- [Kubectl 커맨드 참조](/docs/reference/generated/kubectl/kubectl/)
-- [쿠버네티스 API 참조](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/)
-{{% /capture %}}
\ No newline at end of file
+* [오브젝트 구성을 이용하여 쿠베네티스 관리하기(명령형)](/ko/docs/tasks/manage-kubernetes-objects/imperative-config/)
```
Comment (seokho-son) [inline content/ko/docs/tasks/manage-kubernetes-objects/imperative-command.md]:
```suggestion
* [오브젝트 구성을 이용하여 쿠버네티스 관리하기(명령형)](/ko/docs/tasks/manage-kubernetes-objects/imperative-config/)
```

File: content/ko/docs/tasks/manage-kubernetes-objects/imperative-config.md [hunk 4]
```diff
@@ -10,7 +10,15 @@ weight: 30
 이 문서는 구성파일을 이용하여 어떻게 오브젝트를 정의하고 관리할 수 있는지에 대해 설명한다.
 {{% /capture %}}
 
-{{% capture body %}}
+{{% capture prerequisites %}}
+
+[`kubectl`](/docs/tasks/tools/install-kubectl/)를 설치한다.
```
Comment (seokho-son) [inline content/ko/docs/tasks/manage-kubernetes-objects/imperative-config.md]:
```suggestion
[`kubectl`](/docs/tasks/tools/install-kubectl/)을 설치한다.
```

**Comment by gochist** [comment]
@seokho-son 감사합니다! 리뷰 의견 모두 반영했습니다.

**Comment by seokho-son** [comment]
Thank you ! @gochist 
/lgtm
@ianychoi @ClaudiaJKang, please take a look and approve this PR.

**Comment by gochist** [comment]
PTAL
/assign @ClaudiaJKang @ianychoi

**Pull request merged**